.codeclimate.yml

```diff
@@ -1,6 +1,6 @@
----
 version: "2"
 exclude_patterns:
   - "app/assets/javascripts/arclight/context_navigation.js"
   - "vendor/assets/javascripts/**/*.js"
   - "spec/**/*.rb"
+  - "lib/arclight/traject/ead2_config.rb"
```

Comment:
I'm 👎 on excluding this file entirely; is it possible to be more surgical (or just ignoring codeclimate where it makes sense)?

Reply:
@cbeer Hi Chris - I added this because it was blocking the merge for this PR. What kind of "ignoring" could I do that wouldn't block the PR?

Reply:
@mejackreed - Any suggestions. This PR is now several weeks old as Gary mentioned. :-(